.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,16 @@
+name: Pull Request
+on:
+  pull_request:
+    types: [opened, labeled, synchronize, reopened]
+jobs:
+  RunTests:
+    runs-on: ubuntu-latest
```

Comment:
Does this need to be `runs-on: self-hosted`?

Reply:
I had the same question :)

https://bloomandwild.slack.com/archives/C5PH2LT28/p1661018499053429?thread_ts=1661000241.784649&cid=C5PH2LT28

For public repos we should always run on `ubuntu-latest` since it's free and ulimited.

Additionally by using "ubuntu-latest' it's "outside of our network":

Let's say someone wanted to attack us and we had a public repo that had permissions to run on 'self-hosted' runners. They could submit a PR to one of our public repos and it'd execute inside our staging environment. Where it could theoretically talk to our staging db or do other things we didn't want it to do.

So.. in short.. platform have configured things so that public repos cannot run against self-hosted.